### PR TITLE
Update codethesaurus/docs to use new layout & file structures

### PR DESCRIPTION
**Pull request by cafce25**
This updates the docs to reflect the changes from [codethesaur.us#467](https://github.com/codethesaurus/codethesaur.us/pull/467) and [codethesaur.us#457](https://github.com/codethesaurus/codethesaur.us/pull/457).

### CODE REVIEW

File: docs/thesaurus/index.md [hunk 1]
```diff
@@ -44,47 +59,49 @@ Within the `web/thesauruses/_meta` directory, create the new file you just named
...
 
 ## Add on a New Language
 
 If a programming language doesn't exist yet in Code Thesaurus, you are welcome to add it! There are a few steps to this process.
 
 First, you will need to open up `web/thesauruses/meta_info.json`. Within that file, the first section is for languages. Add in your language to that section in alphabetical order. From there, assign it to a directory name in the JSON structure.
 
-After you save that file, go make a new subdirectory with same name in `web/thesauruses/`. From there, copy one or more of the files in `web/thesauruses/_meta` into the language subdirectory. From there you should be good to implement those features in that concept file!
+After you save that file, go make a new subdirectory with same name in `web/thesauruses/`. From there, copy one or more of the files in `web/thesauruses/_templates` into the language subdirectory. From there you should be good to implement those features in that concept file!
+
```
Comment (geekygirlsarah) [inline docs/thesaurus/index.md:99]: We didn't end up adding a `web/thesauruses/_templates` directory. Is this a leftover from your local machine? The templates are now generated on the website.

File: docs/thesaurus/index.md [hunk 2]
```diff
@@ -44,47 +59,49 @@ Within the `web/thesauruses/_meta` directory, create the new file you just named
...
+For every concept you have to decide:
+* Either provide some example code after the `"code"` label which may be an array to provide multiple code snippets.
+* Or delare that concept as not implemented using `"not-implemented": true`.
```
Comment (geekygirlsarah) [inline docs/thesaurus/index.md:102]: could `delare` be `declare`?

File: docs/thesaurus/index.md [hunk 3]
```diff
@@ -44,47 +59,49 @@ Within the `web/thesauruses/_meta` directory, create the new file you just named
...
+
+In both cases you can add a `"comment"` to explain particuarities about this concept for the language you're describing.
```
Comment (geekygirlsarah) [inline docs/thesaurus/index.md:104]: Another typo, it should be `particularities`

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
Fix the couple of minor things, and I'll merge it in!

File: docs/thesaurus/index.md (hunk 2 above)
Comment (cafce25) [reply docs/thesaurus/index.md]: Uhmm seems I forgot to turn the spellcheck on '-.-

File: docs/thesaurus/index.md (hunk 1 above)
Comment (cafce25) [reply docs/thesaurus/index.md]: Nice catch!

**Comment by cafce25** [comment]
Upon looking at this again it seems to me I had `Adding New Structures to Code Thesaurus` and `Add an Existing Structuer Set to a Language` titles swapped. The last commit fixes that too.

**Review by geekygirlsarah — APPROVED** [review]

**Comment by geekygirlsarah** [comment]
Thanks for fixing these! I'll merge it in.

**Comment by geekygirlsarah** [comment]
This works! I'll merge it in. Thanks for the updates, I appreciate it!

**Pull request merged**